src/client/datascience/jupyter/jupyterNotebook.ts

```diff
@@ -821,6 +823,16 @@ export class JupyterNotebookBase implements INotebook {
             // get a request to update style
             await this.executeSilently(matplobInit, cancelToken);
 
+            const useDark = this.applicationService.activeColorTheme.kind === ColorThemeKind.Dark;
```

Comment:
This code was in the kernel.ts version, but not here so I added it as it looks correct for native theming.